Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:16.13.1-alpine3.15@sha256:a2c7f8ebdec79619fba306cec38150db44a45b48380d09603d3602139c5a5f92 AS builder
+
+WORKDIR /app
+
+RUN [ "yarn", "set", "version", "1.22.17" ]
```

Comment:
We've got yarn downloaded locally in the repo, so you should be able to replace this with something like:

`RUN [ "yarn", "set", "version", ".yarn/releases)/yarn-1.22.17.cjs" ]`

Howeer, you might need to run this command after copying across all the yarn dotfiles

Reply:
using local yarn